src/js/LLMessage.js

```diff
@@ -413,6 +452,9 @@ lm.moveReplay = function(id, numerator) {
     location.href = "./requestReplay.html";
 };
 
+/*
+ * If the received date is past in the past, we will return false.
+ */
```

Comment:
Comments: If the retrieved message's timestamp is not today, it is considered expired.

[Timestamp issue](https://github.com/personium/personium-core/issues/42)